src/s1/t6/n1/exercici2/GenericMethods.java

```diff
@@ -0,0 +1,9 @@
+package s1.t6.n1.exercici2;
+
+public class GenericMethods {
+    public static <T, S, U> void printArguments(T arg1, S arg2, U arg3) {
```

Comment:
When you declare the generic values of the parameters, is it really needed to delcare three different types? Mine seems to be working just fine with declaring only one of those, but idk.

Reply:
It _should_ work fine, as they're generic types. However it denotes that these are 3 different arguments of 3 possibly different types.

Reply:
If you only use `<T>` I guess it means your 3 arguments are of same type. `<T, S, U>` allows for flexibility, because you can do T = S = U for the same argument type or you can have up to 3 different argument types.